ProjectManagmentSystem/src/Modules/Context/AuthContext.tsx

```diff
@@ -10,13 +10,15 @@ export const useAuth = () => {
 
 export default function AuthContextProvider(props: any) {
   const [loginUser, setLoginUser] = useState(null);
-
-
+  let requestHeaders = {Authorization:`Bearer ${localStorage.getItem('token')}`}
```

Comment:
I suggest using axios.create to create an axiosInstance with the baseURL and you can also intercept any api call with axios interceptors and provide the request headers during this interception.
You 'll need to look at : axios.create and axios intereceptors.

Reply:
Done